packages/xstate-react/src/useMachine.ts

```diff
@@ -56,7 +56,7 @@ type RestParams<
         >
     ];
 
-type UseMachineReturn<
+export type UseMachineReturn<
```

Comment:
If we'd export this I think that we should try to think about a better name for this. However, I'm also not sure if we should export this - the builtin React hooks don't export their return types like this, so why we'd do that?

Reply:
yeah you prob right